src/GRA.Domain.Service/VendorCodeService.cs

```diff
@@ -376,10 +391,41 @@ string shipDateString
                                                 issues.Add($"Issue reading details on row {row}: {ex.Message}");
                                             }
                                         }
+
+                                        if (excelReader.GetValue(branchColumnId) != null)
+                                        {
+                                            try
+                                            {
+                                                string branch
+                                                    = excelReader.GetString(branchColumnId);
```

Comment:
I was getting errors on this saying "can't convert double to string", `excelReader.GetValue(branchColumnId).ToString()` seems to work well for getting the value as a string regardless of excel column type.

Reply:
I tested this with a spreadsheet from the vendor and got no errors so I'm going to leave as-is.